test/audit/models/TaxYearSummaryResponseAuditModelSpec.scala

```diff
@@ -23,17 +23,23 @@ import models.incomeSourceDetails.{BusinessDetailsModel, IncomeSourceDetailsMode
 import models.liabilitycalculation.viewmodels.TaxYearSummaryViewModel
 import models.nextUpdates.{NextUpdateModel, NextUpdatesModel, ObligationsModel}
 import org.scalatest.{MustMatchers, WordSpecLike}
+import play.api.i18n.{Messages, MessagesApi}
 import play.api.libs.json.Json
 import play.api.test.FakeRequest
-import testConstants.BaseTestConstants.taxYear
+import testConstants.BaseTestConstants.{app, taxYear}
 import uk.gov.hmrc.auth.core.retrieve.Name
 
 import java.time.LocalDate
 
 class TaxYearSummaryResponseAuditModelSpec extends WordSpecLike with MustMatchers {
 
+  implicit val messages: Messages = app.injector.instanceOf[MessagesApi].preferred(FakeRequest())
```

Comment:
TestSupport Trait gives this already - perhaps use that?

Reply:
Thanks, I've was trying to mix-in with TestSupport in the first place but then we have to inherent conflicting members unfortunately: 
`test
[info] Sass compiling on 1 source(s)
[info] Sass compilation done in 225 ms. 2 resulting css/source map file(s)
[info] compiling 200 Scala sources to /home/aku79/workspace/income-tax-view-change-frontend/target/scala-2.12/test-classes ...
[error] /home/aku79/workspace/income-tax-view-change-frontend/test/audit/models/TaxYearSummaryResponseAuditModelSpec.scala:35:7: class TaxYearSummaryResponseAuditModelSpec inherits conflicting members:
[error]   method convertSymbolToHavePropertyMatcherGenerator in trait MustMatchers of type (symbol: Symbol)(implicit prettifier: org.scalactic.Prettifier, implicit pos: org.scalactic.source.Position)TaxYearSummaryResponseAuditModelSpec.this.HavePropertyMatcherGenerator  and
[error]   method convertSymbolToHavePropertyMatcherGenerator in trait Matchers of type (symbol: Symbol)(implicit prettifier: org.scalactic.Prettifier, implicit pos: org.scalactic.source.Position)TaxYearSummaryResponseAuditModelSpec.this.HavePropertyMatcherGenerator
[error] (Note: this can be resolved by declaring an override in class TaxYearSummaryResponseAuditModelSpec.);
[error]  other members with override errors are: equal, key, value, a, an, theSameInstanceAs, regex, <, >, <=, >=, definedAt, oneOf, oneElementOf, atLeastOneOf, atLeastOneElementOf, noneOf, noElementsOf, theSameElementsAs, theSameElementsInOrderAs, only, inOrderOnly, allOf, allElementsOf, inOrder, inOrderElementsOf, atMostOneOf, atMostOneElementOf, thrownBy, message, all, atLeast, every, exactly, no, between, atMost, the, convertToRegexWrapper, of
[error] class TaxYearSummaryResponseAuditModelSpec extends WordSpecLike with MustMatchers with TestSupport {
[error]       ^
[error] one error found
[error] (Test / compileIncremental) Compilation failed `

Reply:
Done, I've just pushed the fix, thanks